proto/logbackuppb.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
```

Comment:
maybe add the license

Reply:
It seems that other proto files don't have license header too.🤔